Assets/PatchKit Patcher/Scripts/AppData/Local/MapHashExtractedFiles.cs

```diff
@@ -11,21 +11,22 @@ public MapHashExtractedFiles()
             MapHash = new Dictionary<string, string>();
         }
         
-        public string Add(string path)
+        public string GetNameHash(string path)
```

Comment:
Not very fortunate name, because `Get` suggest no side effect. I would rename it to `CalculateHashFromPath()` or `Hash()`.

By the way, how often this function receives the same path names? MD5 calculation is pretty fast. If that's 2-3 times at most, I wouldn't use internal hashmap to cache it at all.

Reply:
Done